app/controllers/authentications/identities_controller.rb

```
@@ -8,7 +8,7 @@ def new
 
     def create
       identity = Identity.new(identity_params.merge(email: current_user.email))
-      if identity.save && current_user.create_auth_for_identity(identity)
+      if identity.save! && current_user.create_auth_for_identity(identity)
```

@mafarain This is what I have talked to you: don't refactor such things, change only that pieces of code which can SILENT errors, this piece of code doesn't work so.